src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Would it be better to use private modifier here? Or is there a particular reason to use protected?